### ISSUE TITLE
Add more validation metrics: MSE, MAE, R2

### ISSUE THREAD

**Issue by KarolinaChalupova** [KarolinaChalupova/DiplomaThesis#7]

**Comment by KarolinaChalupova** [comment]
I now have RMSE and MAE, tensordlow_addons RSquare does not work with model.compile as a metric

**Comment by KarolinaChalupova** [comment]
I can implement R2 myself by extending the tf.keras.metrics class myself: 
https://towardsdatascience.com/three-ways-to-use-custom-validation-metrics-in-tf-keras-tf2-bb9c40a3076
